parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs

```diff
@@ -593,3 +600,80 @@ where
 		sp_std::boxed::Box::new(Self::asset_id(asset_id))
 	}
 }
+
+/// All configuration related to bridging
+pub mod bridging {
+	use super::*;
+	use assets_common::{matching, matching::*};
+	use sp_std::collections::btree_set::BTreeSet;
+	use xcm_builder::{NetworkExportTable, UnpaidRemoteExporter};
+
+	parameter_types! {
```

Comment:
Might be nice to name these types to westend like you did for Kusama/Polkadot. Or are these types different?

Reply:
AssetHubWestend changes are better reverted